### PR TITLE
Add a quick guide

### PR DESCRIPTION
**Pull request by mmattel**
Add a quick up and running guide.

Already on staging.

### CODE REVIEW

File: modules/ROOT/pages/deployment/index.adoc [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-:toclevels: 1
 
 :description: There are several deployment options for ownCloud Infinite Scale (oCIS) available. Consider which one serves your purposes best. Generally speaking, you can decide between using Docker containers or manually set up a regular server.
 
 // https://owncloud.dev/ocis/deployment/
 
 {description}
 
-* If you only want to play around and see what oCIS has to offer, we suggest the quick and easy xref:deployment/binary/binary-setup.adoc[Binary Setup] section.
+* The section xref:general/general-info.adoc[General Info] summarizes general and important information which is not dependent on the type how you install oCIS. Any setup refers to this general information as prerequisit knowledge without explicit reference.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/index.adoc:10]:
```suggestion
* The section xref:general/general-info.adoc[General Info] summarizes general and important information which is not dependent on the way that you install oCIS. Any setup refers to this general information as prerequisite knowledge.
```

File: modules/ROOT/pages/deployment/index.adoc [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 
-* In case you have already decided on a Docker instance with one or more containers, checkout the xref:deployment/container/container-setup.adoc[Container Setup] section.
+* If you only want to play around and see what oCIS has to offer, we suggest the quick and easy xref:deployment/binary/binary-setup.adoc[Binary Setup] section.
 
-* xref:deployment/security/security.adoc[Securing oCIS] should be a primary concern after you've set up your server.
+* In case you have already decided on a container instance with one or more containers, checkout the xref:deployment/container/container-setup.adoc[Container Setup] section.
```
Comment (phil-davis) [inline modules/ROOT/pages/deployment/index.adoc:14]:
```suggestion
* If you have already decided on a container instance with one or more containers, checkout the xref:deployment/container/container-setup.adoc[Container Setup] section.
```

File: modules/ROOT/pages/quickguide/quickguide.adoc [hunk 3]
```diff
@@ -0,0 +1,37 @@
+= Quick up and Running Guide
+:toc:right
+:downloadpage_ocis_url: https://download.owncloud.com/ocis/ocis/stable/
+
+:description: This quick up and running guide provides brief copy and paste commands to have an oCIS instance ready for hands on. Consider reading the documentation for details necessary when you dive down and plan to go productive.
```
Comment (phil-davis) [inline modules/ROOT/pages/quickguide/quickguide.adoc:5]:
```suggestion
:description: This quick up and running guide provides brief copy and paste commands to have an oCIS instance ready for hands on use. Read the documentation for details necessary when you dive down and plan to deploy into production.
```

File: modules/ROOT/pages/quickguide/quickguide.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+== Introduction
+
+{description}
+
+Using this guide will setup a minimal configured environment up and running but can by nature not go into the depth. Note that the binary used is based on `linux-arm-64bit`.
```
Comment (phil-davis) [inline modules/ROOT/pages/quickguide/quickguide.adoc:11]:
```suggestion
Using this guide will get a minimal configured environment up and running but can by nature not go into depth. Note that the binary used is based on `linux-arm-64bit`.
```

File: modules/ROOT/pages/quickguide/quickguide.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+We recommend reading the xref:deployment/index.adoc[Deployment] section of the documentation as next step for more information on setup and configuration possibilities.
```
Comment (phil-davis) [inline modules/ROOT/pages/quickguide/quickguide.adoc:13]:
```suggestion
We recommend reading the xref:deployment/index.adoc[Deployment] section of the documentation as the next step for more information on setup and configuration possibilities.
```

File: modules/ROOT/pages/quickguide/quickguide.adoc [hunk 6]
```diff
@@ -0,0 +1,37 @@
+= Quick up and Running Guide
+:toc:right
+:downloadpage_ocis_url: https://download.owncloud.com/ocis/ocis/stable/
+
+:description: This quick up and running guide provides brief copy and paste commands to have an oCIS instance ready for hands on use. Read the documentation for details necessary when you dive down and plan to deploy into production.
```
Comment (EParzefall) [inline modules/ROOT/pages/quickguide/quickguide.adoc:5]:
```suggestion
:description: This quick guide provides copy and paste commands to have an oCIS instance up and running within minutes for hands on use. Read the documentation for details before you dive down and plan to deploy into production.
```

File: modules/ROOT/pages/quickguide/quickguide.adoc [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+[source,bash]
+----
+OCIS_INSECURE=true \
+PROXY_HTTP_ADDR=0.0.0.0:9200 \
+OCIS_URL=https://localhost:9200 \
+ocis server
+----
+
+After installing and starting oCIS, you can now access the web UI via `\https://localhost:9200`.
```
Comment (EParzefall) [inline modules/ROOT/pages/quickguide/quickguide.adoc:37]:
```suggestion
After installing and starting oCIS, you can access the web UI via `\https://localhost:9200`.
```

**Review by EParzefall — APPROVED** [review]
minor language fixes

**Pull request merged**